pyproject.toml

```
@@ -7,7 +7,7 @@ readme = "README.md"
 
 
 [tool.poetry.dependencies]
-python = "==3.12.2"
+python = "^3.12.2"
```

Let's make this explicit so we're sure things match:

```suggestion
python = "==3.12.3"
```

---

Nevermind, we're going to try as is!